packages/server/src/adapters/aws-lambda/index.ts

```diff
@@ -52,9 +52,23 @@
 }
 function getPath(event: APIGatewayEvent) {
   if (isPayloadV1(event)) {
+    const matches = event.resource.matchAll(/\{(.*?)\}/g);
+    for (const match of matches) {
+      const group = match[1];
+      if (group.includes('*') && event.pathParameters) {
+        return event.pathParameters[group.replace('*', '')] || '';
```

Comment:
Interesting that it took offense to `group.replace('*', '')` and not `group.replace('+', '')` :laughing:

Reply:
I wonder if you need to escape both? Doesn't look like it's harmful: 
![image](https://user-images.githubusercontent.com/6134511/178454328-e7b3c21c-559b-483e-875d-cea3e3d3481f.png)

Reply:
Eslint doesn't like that. Because this is a trusted input (i.e. user cannot use this to execute an unwanted trpc query) I would suggest that this is a false positive and my code is fine